nixos/modules/services/logging/pushlog.nix

```
@@ -0,0 +1,171 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.pushlog;
+  unitType = types.submodule {
+    options = {
+      match = mkOption {
+        type = types.str;
+        default = ".*";
+      };
+      priorities = mkOption {
+        type = types.listOf types.int;
+        default = [
+          0
+          1
+          2
+          3
+          4
+          5
+          6
+        ];
+      };
+      include = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+      };
+      exclude = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+      };
+    };
+  };
+in
+{
+  options.services.pushlog = {
+    enable = mkEnableOption "Enable pushlog service to forward journald logs to Pushover";
```

"Enable" is already included as part of the description, so what you have here is essentially "Enable Enable..."

```suggestion
    enable = mkEnableOption "pushlog service to forward journald logs to Pushover";
```